CONTRIBUTING.md

```diff
@@ -7,9 +7,9 @@
 
 ## Setup Development (developed using Node v4.3.1)
 
- - `npm install -g gulp-cli`
- - `npm install -g karma-cli`
```

Comment:
reminder: do a `npm remove -g gulp-cli` and `npm remove -g karma-cli` before testing these.